### ISSUE TITLE
Pyrewood Village spams a howling sound way too often.

### ISSUE THREAD

**Issue by Kaporalhart** [slowtorta/turtlewow-bug-tracker#1421]
**Please, provide a clear description what the bug is:**

In Pyrewood Village, in the Silverpine Forest, a howling sound is heard way too many times. It can be heard sveeral times per second sometimes.

**Steps to reproduce the behavior with as much detail as possible:**

1. Just get to Pyrewood village. The villagers were in their wolf forms at that moment.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

Nothing was making the sound as far as i know. it was just around pyrewood village, and could be heard from the entrance of Shadowfang Keep.

**Expected behavior. Describe how it should work:**

Maybe, we could hear it every once in a while ? But it was being spammed instead.

**Comment by AyleeJenn** [comment]
I have looked into this with my group when we were at that level and we believe we found a pattern to this. The wolf howl seems to happen whenever one of the pyrewood npcs is rendered/loaded into your client. And that depends on the distance at which your client loads them for you to see.

That would explain the howl spamming since some of the npcs are rather clustered and when a bunch of them loads in, ... howling commences.
